tasks/vendoring.py

```diff
@@ -0,0 +1,213 @@
+""""Vendoring script, python 3.5 needed"""
+
+from pathlib import Path
+import re
+import shutil
+
+import invoke
+
+TASK_NAME = 'update'
+
+FILE_WHITE_LIST = (
+    'Makefile',
+    'vendor.txt',
+    '__init__.py',
+    'README.rst',
+)
+
+
+SPECIAL_CASES = (
```

Comment:
What about turning these into actual diff files stored alongside this (``tasks/vendoring/__init__.py`` and ``tasks/vendoring/patches/*.diff``) and restoring using ``difflib.restore()``? That seems like it'd be easier to manage then baking them into the file itself.